contracts/interchain-token-service/src/abi.rs

```diff
@@ -391,7 +448,7 @@ mod tests {
         goldie::assert_json!(encoded);
 
         for original in cases {
-            let encoded = original.clone().abi_encode(&env);
+            let encoded = original.clone().abi_encode(&env).unwrap();
```

Comment:
```suggestion
            let encoded = assert_ok!(original.clone().abi_encode(&env));
```
use assert_ok from axelar-soroban-std to emphasize that we're checking that this function doesn't error